src/routes/index.tsx

```diff
@@ -130,7 +131,8 @@ export default function Home() {
             <Field name="emoji">
               {(field, props) => (
                 <>
-                  <label>trigger emoji</label>
+                  <label class="mb-0">trigger emoji</label>
+                  <label class="text-sm font-normal mt-0 opacity-75">Damus uses 🤙 by default. Other emojis will not trigger zaps</label>
```

Comment:
How about:

Which reaction emoji do want to trigger zaps? Damus uses 🤙 by default.